service-python/tests/assessclaim/test_queues.py

```diff
@@ -13,11 +12,8 @@
 from assessclaimdc6602.src.lib.main import assess_asthma as main6602
 from assessclaimdc6602v2.src.lib import queues as q6602v2
 from assessclaimdc6602v2.src.lib.main import assess_asthma as main6602v2
-from assessclaimdc7101.src import logging_setup
 from assessclaimdc7101.src.lib import queues as q7101
 
-logging_setup.set_format()
```

Comment:
Looks like this wasn't being used and it was leading to import issues so I removed it since it was causing issues with the import paths